client/state/themes/selectors.js

```diff
@@ -96,6 +96,17 @@ export const getThemesForQuery = createSelector(
 );
 
 /**
+ * Returns last query used.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {String}         Last quer
+ */
+export function getLastQuery( state, siteId ) {
+	return get( state.themes.lastQuery, siteId, '' );
```

Comment:
Hmm, `query` is an object, so we can't fall back to `''` but rather to `{}`. (Same for JSDoc `@return` type.)